docs/docsite/rst/user_guide/vault.rst

```diff
@@ -424,6 +429,58 @@ When using :ref:`ansible-vault` commands that encrypt content (:ref:`ansible-vau
 only one vault-id can be used.
 
 
+.. _vault_securing_editor:
+Securing your editor
+````````````````````
+
+As noted above, the vault system relies on your configured ``$EDITOR`` which can be a source of disclosures, most editors have conviniences to prevent loss of data, but this normaly relies
```

Comment:
```suggestion
As noted above, the vault system relies on your configured ``$EDITOR`` which can be a source of disclosures. Most editors have ways to prevent loss of data, but these normally rely
```